.github/workflows/release.yml

```diff
@@ -0,0 +1,38 @@
+# When a tag is pushed, create a tag-named release with the brokerpak and Helm chart in it 
+name: 'release'
+on:
+  push:
+    tags:
+    - '*'
+
+jobs:
+  release:
+    name: 'Release'
+    runs-on: ubuntu-latest
+
+    # Use the Bash shell regardless whether the GitHub Actions runner is ubuntu-latest, macos-latest, or windows-latest
+    defaults:
+      run:
+        shell: bash
+
+    steps:  
+    - name: Check out repository
+      uses: actions/checkout@v2
+      with: 
+        fetch-depth: '0'
+
+    - name: Build the brokerpak
+      run: make build
+
+    - name: Rename the brokerpak file based on the tag
+      run: mv datagov-services-pak-current.brokerpak datagov-services-pak-${GITHUB_REF#refs/*/}.brokerpak
+
+    - name: Create helm chart tarball
+      run: cd terraform/provision && tar czvf ../../solr-crd.tar.gz solr-crd
```

Comment:
We should move this to the Makefile so it's straightforward to build the release locally (and in case we want to sign it in the future). Also add the tar.gz to .gitignore.